### ISSUE TITLE
Add payment characters string in addition to the QR

### ISSUE THREAD

**Issue by jpl-btc** [getAlby/pos#24]
It would be nice to have the payment string and be able to copy it to the clipboard by simply clicking on it with a finger or with a mouse click, for mobile or desktop purposes, just below the qr code.

![image](https://github.com/user-attachments/assets/832ef8c8-3121-43eb-a38e-b534e6e74151)

**Comment by bumi** [comment]
I don't think this is the intended use. The PoS you will never pay yourself from another wallet. why do you need this?

**Comment by jpl-btc** [comment]
I want to use the pos to share the expenses of some empanadas we will eat in a friend's gathering and send the invoice through whatsapp
